src/components/application_manager/src/commands/mobile/on_vehicle_data_notification.cc

```diff
@@ -56,13 +56,24 @@ void OnVehicleDataNotification::Run() {
   std::vector<smart_objects::SmartObject> appSO;
 
   const VehicleData& vehicle_data = MessageHelper::vehicle_data();
+  const smart_objects::SmartObject incoming_msg_params = (*message_)[strings::msg_params];
```

Comment:
@KVGrygoriev Why do you copy it? May we use const ref?

Reply:
@AByzhynar, good point! See 41aa1c3b2744ac4a3d72c905a500c5b7b1e45ab3